### PR TITLE
Update vSNP tools

### PR DESCRIPTION
**Pull request by gregvonkuster**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/vsnp/vsnp_add_zero_coverage.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-<tool id="vsnp_add_zero_coverage" name="vSNP: add zero coverage" version="@WRAPPER_VERSION@.2+galaxy0" profile="@PROFILE@">
+<tool id="vsnp_add_zero_coverage" name="vSNP: add zero coverage" version="@TOOL_VERSION@.2+galaxy0" profile="@PROFILE@">
```
Comment (bgruening) [inline tools/vsnp/vsnp_add_zero_coverage.xml:1]: @gregvonkuster you need to bump all he galaxy0 to galaxy1, isn't it?

File: tools/vsnp/vsnp_statistics.xml [hunk 2]
```diff
@@ -1,202 +1,124 @@
-<tool id="vsnp_statistics" name="vSNP: statistics" version="@WRAPPER_VERSION@.1+galaxy0" profile="@PROFILE@">
+<tool id="vsnp_statistics" name="vSNP: statistics" version="@TOOL_VERSION@.2+galaxy0" profile="@PROFILE@">
```
Comment (bgruening) [inline tools/vsnp/vsnp_statistics.xml:1]: what is this .2 doing? the galaxy0 is supposed to be the version increment.

Comment (gregvonkuster) [reply tools/vsnp/vsnp_statistics.xml]: Ah, sorry!  Don't know how those got so screwed up.  Fixed now.

**Comment by gregvonkuster** [comment]
Looks like all tests pass - just file size linting is a problem.
```
Files larger than 500k found
tools/vsnp/test-data/BCG_Pasteur_Unknown_FR_SRR8886989.vcf
tools/vsnp/test-data/BCG_Tokyo_Unknown_JP_DRR029468.vcf
tools/vsnp/test-data/Mcap_Deer_DE_SRR650221.fastq.gz
Error: Process completed with exit code 1.
```

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks Greg!

**Pull request merged**